UI/signup.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+  <meta name="description" content="Andela Challenge one">
+  <meta name="keywords" content="HTML,CSS,JavaScript">
+  <meta name="author" content="Irene Nyakate">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<style>
```

Comment:
It is good practice to separate the css from the html. Create a .css file and move the css to that file and then reference that file in the style tag. I believe Mellon has done it before. So contact her if you have any problems.